webpack.config.js

```diff
@@ -0,0 +1,35 @@
+// const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: ['./src/index.js',
+    './src/style.css',
+  ],
+
+  devServer: {
+    static: './dist',
+  },
+
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+    new MiniCssExtractPlugin({
+      filename: '[name].css',
+    }),
+  ],
+
+  //   optimization: {
+  //     runtimeChunk: 'single',
+  //   },
```

Comment:
```suggestion
```

Reply:
Kindly consider removing any comments that don't add anything to your code to comply with KISS and YAGNI principles.